src/DotNetLightning.Core/Serialize/Features.fs

```diff
@@ -230,6 +224,29 @@ type FeatureBit private (bitArray) =
     override this.ToString() =
         this.BitArray.PrintBits()
         
+    member this.SetFeatureBit(index: int, on: bool): unit =
```

Comment:
Why not take `Feature` type as an argument? isn't that safer because the API consumer cannot pass a wrong index number?

Reply:
I mean both `Feature` and `FeaturesSupport` as an argument.

Reply:
Okay, I've changed it.